go.mod

```diff
@@ -32,6 +32,7 @@ require (
 )
 
 require (
+	github.com/djherbis/times v1.5.0
```

Comment:
Go pkg to support file time attributes (ctime, mtime, atime) for most platforms.